frontend/src/metabase/parameters/components/ParametersList/ParametersList.tsx

```diff
@@ -66,8 +72,7 @@ function ParametersList({
       setEditingParameter={setEditingParameter}
       setValue={
         setParameterValue &&
-        (value =>
-          setParameterValue(valuePopulatedParameter.id, value, dashboard?.id))
+        ((value: any) => setParameterValue(valuePopulatedParameter.id, value))
```

Comment:
I suppose the `value` here can be of multiple types?

Reply:
yep, I'll actually change this to `unknown` since the `setParameterAction` expects:
```
(parameterId: ParameterId, value: unknown)
```